store/evm_aux/store.go

```diff
@@ -102,32 +124,28 @@ func (s *EvmAuxStore) SaveChildTxRefs(refs []ChildTxRef) error {
 	if len(refs) == 0 {
 		return nil
 	}
-
-	tran, err := s.db.OpenTransaction()
-	if err != nil {
-		return errors.Wrap(err, "failed to open tx in EvmAuxStore")
-	}
-	defer tran.Discard()
-
+	batch := s.db.NewBatch()
+	defer batch.Close()
 	for _, ref := range refs {
-		tran.Put(util.PrefixKey(txRefPrefix, ref.ParentTxHash), ref.ChildTxHash, nil)
+		batch.Set(util.PrefixKey(txRefPrefix, ref.ParentTxHash), ref.ChildTxHash)
 	}
-
-	if err := tran.Commit(); err != nil {
-		return errors.Wrap(err, "failed to commit tx in EvmAuxStore")
-	}
-
+	batch.Write()
 	return nil
 }
 
 // GetChildTxHash looks up the EVM tx hash that corresponds to the given Tendermint tx hash.
-func (s *EvmAuxStore) GetChildTxHash(parentTxHash []byte) ([]byte, error) {
-	return s.db.Get(util.PrefixKey(txRefPrefix, parentTxHash), nil)
+func (s *EvmAuxStore) GetChildTxHash(parentTxHash []byte) []byte {
+	return s.db.Get(util.PrefixKey(txRefPrefix, parentTxHash))
 }
 
-func (s *EvmAuxStore) DB() *leveldb.DB {
+func (s *EvmAuxStore) DB() dbm.DB {
 	return s.db
 }
+
+func (s *EvmAuxStore) Commit() {
+	s.store.Commit()
+}
+
 func (s *EvmAuxStore) ClearData() {
-	os.RemoveAll(EvmAuxDBName)
+	os.RemoveAll(legacyEvmAuxDBName)
```

Comment:
Lets get rid of this function, call `os.RemoveAll` in the tests - where necessary.

Reply:
Done